Sources/LeafKit/LeafParser.swift

```diff
@@ -38,6 +38,11 @@ extension TagDeclaration {
                     var buffer = ByteBufferAllocator().buffer(capacity: 0)
                     buffer.writeString(st)
                     return .raw(buffer)
+                case .keyword(let kw) :
+                    guard kw.isBooleanValued else { fallthrough }
+                    var buffer = ByteBufferAllocator().buffer(capacity: 0)
```

Comment:
Use `kw.rawValue.count` as the capacity.

Reply:
Same reply as above